packages/grafana-ui/src/components/TimePicker/TimeZonePicker.tsx

```diff
@@ -63,7 +63,7 @@ interface SelectableZoneGroup extends SelectableValue<string> {
 const useTimeZones = (includeInternal: boolean): SelectableZoneGroup[] => {
   const now = Date.now();
 
-  return getTimeZoneGroups(includeInternal).map((group: GroupedTimeZones) => {
+  const timeZoneGroups = getTimeZoneGroups(includeInternal).map((group: GroupedTimeZones) => {
```

Comment:
I like the refactoring but I think we need to make sure that this hasn't to big of a performance impact.

Reply:
This was a tricky one and I've checked and saw why the memoization needed for this but this only run on init and on change which didn't feel laggy or something but I only checked this in the configuration/preferences page. Is it used somewhere else?

Reply:
Yes, in the TimePicker.

<img width="687" alt="Skärmavbild 2020-10-08 kl  16 56 03" src="https://user-images.githubusercontent.com/172951/95476083-2ae7c780-0987-11eb-9de5-bd5932324e1f.png">

Reply:
dah 😅

Reply:
Checked it and I didn't feel any performance issue there.